projects/vanilla/src/app/search/search.component.ts

```diff
@@ -18,7 +18,8 @@ import { BsFacetDate } from '@sinequa/analytics/timeline';
 @Component({
   selector: 'app-search',
   templateUrl: './search.component.html',
-  styleUrls: ['./search.component.scss']
+  styleUrls: ['./search.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
Maybe using ::ng-deep in the CSS could have less potential side-effects?

Reply:
not needed here, the `ViewEncapsulation.None` do the job and in the `scss` file, only css style about `seach-component` is set